daprdocs/content/en/developing-applications/debugging/bridge-to-kubernetes.md

```diff
@@ -0,0 +1,26 @@
+---
+type: docs
+title: "Bridge to Kubernetes support for Dapr services"
+linkTitle: "Bridge to Kubernets"
+weight: 300
+description: "Debug Dapr apps locally which still connected to your Kubernetes cluster"
+---
+
+Bridge to Kubernetes allows you to run and debug code on your development computer, while still connected to your Kubernetes cluster with the rest of your application or services. This type of debugging is often called *local tunnel debugging*. 
+
+{{< button text="Learn more about Bridge to Kubernetes" link="https://code.visualstudio.com/docs/containers/bridge-to-kubernetes" >}}
+
+## Debug Dapr apps
+
+Bridge to Kubernetes supports debugging Dapr apps on your machine, while still having them interact with the services and applications running on your Kubernetes cluster. This example showcases Bridge to Kubernetes enabling a developer to debug the [distributed calculator quickstart](https://github.com/dapr/quickstarts/tree/master/distributed-calculator):
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/rxwg-__otso" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

Comment:
The video is good, but would it make sense to provide step by step instructions as well (similar to what exists for `Debug daprd on Kubernetes`)?
If it helps, we can provide you with step by step instructions, or directly send a PR in this repo with the suggested changes.

Reply:
Feel free to send step-by-step instructions, but I personally do not think we should block on it. This video is good!